Library/ViewModels/FindFriendsFriendFollowCellViewModel.swift

```diff
@@ -135,6 +143,39 @@ public final class FindFriendsFriendFollowCellViewModel: FindFriendsFriendFollow
       )
       .skipRepeats()
 
+    self.followButtonAccessibilityLabel = name
+      .map {
+        localizedString(
+          key: "Follow_friend_name",
+          defaultValue: "Follow %{friend_name}",
+          count: nil,
+          substitutions: ["friend_name": $0])
+      }
+
+    self.unfollowButtonAccessibilityLabel = name
+      .map {
+        localizedString(
+          key: "Stop_following_friend_name",
+          defaultValue: "Stop following %{friend_name}",
+          count: nil,
+          substitutions: ["friend_name": $0])
+      }
+
+    let followedText = localizedString(
+      key: "Followed",
+      defaultValue: "Followed",
+      count: nil,
+      substitutions: [:])
+
+    let notFollowedText = localizedString(
+      key: "Not_followed",
+      defaultValue: "Not followed",
+      count: nil,
+      substitutions: [:])
+
+    self.cellAccessibilityLabel = Signal.combineLatest(self.name, self.projectsBackedText, isFollowed)
+      .map { "\($0.0), \($0.1). " + ($0.2 ? followedText : notFollowedText) }
```

Comment:
Once your strings are merged and these are functions on `Strings` we would probably just move them to be used in the ternary operator rather than variables declared in scope, but for now this improves readability 👍